src/Schema/CockroachSchemaState.php

```diff
@@ -33,7 +33,7 @@ public function dump(Connection $connection, $path): void
         });
 
         if ($statements !== []) {
-            $file .= PHP_EOL . $statements[0]['query'];
+            $file .= PHP_EOL . $statements[0]['query'] .(ends_with(trim($statements[0]['query']), ';') ? '' : ';');
```

Comment:
ends_with will never work as the helper isn't installed in all applications with Laravel.

Reply:
```suggestion
            $file .= PHP_EOL . Str::finish($statements[0]['query'], ';');
```

And import `use Illuminate\Support\Str;`